blockservice/blockservice.go

```diff
@@ -169,6 +171,7 @@ func (s *blockService) AddBlocks(bs []blocks.Block) error {
 	}
 
 	for _, o := range toput {
+		log.Event(context.TODO(), "BlockService.BlockAdded", o.Cid())
```

Comment:
Nit: I like naming the events based on the method name, e.g. `BlockService.AddBlocks`.

Reply:
I think in this case it makes sense to have custom named events as they are mostly for functional reasons. It would also mean having 4 different events instead of 2.